builtins/amp-video.js

```diff
@@ -45,11 +52,20 @@ export function installVideo(win) {
             'No "poster" attribute has been provided for amp-video.');
       }
 
+      videoManagerForDoc(this.win.document).register(this);
+
       // Disable video preload in prerender mode.
       this.video_.setAttribute('preload', 'none');
       this.propagateAttributes(['poster', 'controls'], this.video_);
       this.applyFillContent(this.video_, true);
       this.element.appendChild(this.video_);
+
+      this.element.dispatchCustomEvent(VideoEvents.BUILT);
```

Comment:
But if we do use `videoManager.register(this)` here, then why do we need `BUILT` event? Registration is essentially equivalent to being built.